lib/cli/src/frameworks/common/0.Introduction.stories.mdx

```diff
@@ -0,0 +1,194 @@
+    font-size: 13px;
+    line-height: 20px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+  }
+
+  .tip-wrapper code {
+    font-size: 12px;
+    display: inline-block;
+  }
+
+  
+`}</style>
+
+# Welcome to Storybook
+
+Storybook helps you build UI components in isolation from your app's business logic, data, and context.
+That makes it easy to develop hard-to-reach states. Save these UI states as **stories** to revisit during development, testing, or QA.
+
+Browse example stories now by navigating to them in the sidebar.
+View their code in the `src/storybook-examples` directory to learn how they work.
+We recommend building UIs with a [**component-driven**](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) process starting with atomic components and ending with pages.
+
+<div class="subheading">Configure</div>
+
+<div class="link-list">
+  <a class="link-item" href="https://storybook.js.org/docs/presets" target="_blank">
+    <img src={Plugin} alt="plugin" />
+    <span>
+      <strong>Presets for popular tools</strong>
+      Easy setup for TypeScript, SCSS and more.
+    </span>
+  </a>
+  <a class="link-item" href="https://storybook.js.org/docs/webpack" target="_blank">
+    <img src={StackAlt} alt="Build" />
+    <span>
+      <strong>Build configuration</strong>
+      How to customize webpack and Babel
+    </span>
+  </a>
+  <a class="link-item" href="https://storybook.js.org/docs/styling" target="_blank">
+    <img src={Colors} alt="colors" />
+    <span>
+      <strong>Styling</strong>
+      How to load and configure CSS libraries
+    </span>
+  </a>
+  <a class="link-item" href="https://storybook.js.org/docs/data" target="_blank">
+    <img src={Flow} alt="flow" />
+    <span>
+      <strong>Data</strong>
+      Providers and mocking for data libraries
+    </span>
+  </a>
+</div>
+
+<div class="subheading">Learn</div>
+
+<div class="link-list">
+  <a class="link-item" href="https://storybook.js.org/docs" target="_blank">
+    <img src={Repo} alt="repo" />
+    <span>
+      <strong>Storybook documentation</strong>
+      Configure, customize, and extend
+    </span>
+  </a>
+  <a class="link-item" href="https://www.learnstorybook.com" target="_blank">
+    <img src={Direction} alt="direction" />
+    <span>
+      <strong>In-depth guides</strong>
+      Best practices from leading teams
+    </span>
+  </a>
+  <a class="link-item" href="https://github.com/storybookjs/storybook" target="_blank">
+    <img src={Code} alt="code" />
+    <span>
+      <strong>GitHub project</strong>
+      View the source and add issues
+    </span>
+  </a>
+  <a class="link-item" href="https://discord.gg/UUt2PJb" target="_blank">
+    <img src={Comments} alt="comments" />
+    <span>
+      <strong>Discord chat</strong>
+      Chat with maintainers and the community
+    </span>
+  </a>
+</div>
+
+<div class="tip-wrapper">
+  <span class="tip">Tip</span>Edit the Markdown in <code>src/storybook-examples/welcome.mdx</code>
```

Comment:
Here it says `src/storybook-examples/welcome.mdx` although the generated file is `stories/0.Introduction.stories.mdx`

Reply:
That was my mistake! @tmeasday why is the file called `0.Introduction`? Is it so Storybook shows it as the first story without having extra configuration in `main.js`?

Reply:
Yes, I believe so. However, in the example it's not being shown first 🤔

![image](https://user-images.githubusercontent.com/1671563/86609028-9b0f3780-bfab-11ea-99f3-bc44c18d6928.png)

Reply:
> @tmeasday why is the file called 0.Introduction?  Is it so Storybook shows it as the first story without having extra configuration in main.js?

That was originally why, yeah. Can we just change the stories field to import `mdx` first? Or is that too complex?

Reply:
@tmeasday I think it looks weird to have 0.name when I first install. I'd rather the default story import be something reasonable like showing MDX files first.

Reply:
I have experienced quite some colleagues keeping this nomenclature in order to have properly ordered stories, and of course it all starts because the template does it.
For the ones that want to have a simpler way of organizing the stories (per category is something I see a lot), notice people creating a custom sorting mechanism like this one: https://github.com/ing-bank/lion/blob/master/.storybook/preview.js#L30

Sorting stories should be something simple by default and if people want to do something more complex they should be free to do so, IMO.